frontend/src/choreoComponents/ChoreoCard.tsx

```diff
@@ -0,0 +1,47 @@
+import {Choreo} from "../models/Choreo";
+import React from "react";
+import {Box, Button, Card, CardActions, CardContent, Typography} from "@mui/material";
+import RemoveRedEyeIcon from "@mui/icons-material/RemoveRedEye";
+import {useNavigate} from "react-router-dom";
+
+type ChoreoProps = {
+    choreo: Choreo
+}
+export default function ChoreoCard(props: ChoreoProps) {
+
+    const navigate = useNavigate()
+
+    return (
+        <div className="choreo-card">
+            <Box sx={{ boxShadow: 5, width: 500, mb: 2, padding: 1}}>
+                <h2>{props.choreo.name.toUpperCase()}</h2>
+                {props.choreo.choreoMoves.map((move) =>
+                    (<Card className='choreo-move' style={{backgroundColor: "#1B1E24",
```

Comment:
beim mapping im frontend sollte man das key-attribut bei den komponenten setzen, die durch das mapping entstehen, sonst kann es zu unerwartetem verhalten kommen

Reply:
zum spicken kannst du in deine ChoreoGallery.tsx schauen

Reply:
erledigt :-)